### PR TITLE
Cut number of volumes in half

### LINKED ISSUE #88: Eliminate ebs-backed volumes for logs
It will allow us to run twice as many vchains per single instance.

### PR DESCRIPTION
**Pull request by netoneko**
Fixes #88

### CODE REVIEW

**Comment by amir-arad** [comment]
@netoneko did you test it?

**Comment by netoneko** [comment]
@amir-arad I did test it couple of months ago

**Comment by netoneko** [comment]
I have addressed the comments

File: strelets/adapter/swarm_volumes.go [hunk 1]
```diff
@@ -63,6 +63,14 @@ func (d *dockerSwarmOrchestrator) provisionVolume(ctx context.Context, volumeNam
 	}, nil
 }
 
+func (d *dockerSwarmOrchestrator) provisionNetworkVolume(ctx context.Context, volumeName string, target string, maxSizeInGb int) (mount.Mount, error) {
```
Comment (amir-arad) [inline strelets/adapter/swarm_volumes.go]: this function is expendable. please remove it

File: strelets/adapter/swarm_volumes.go [hunk 2]
```diff
@@ -63,6 +63,14 @@ func (d *dockerSwarmOrchestrator) provisionVolume(ctx context.Context, volumeNam
...
+	return d.provisionVolume(ctx, volumeName, target, maxSizeInGb, d.options)
+}
+
+func (d *dockerSwarmOrchestrator) provisionLocalVolume(ctx context.Context, volumeName string, target string, maxSizeInGb int) (mount.Mount, error) {
```
Comment (amir-arad) [inline strelets/adapter/swarm_volumes.go]: this function is expendable. please remove it

File: strelets/adapter/swarm_volumes.go (hunk 1 above)
Comment (netoneko) [reply strelets/adapter/swarm_volumes.go]: I would rather have two functions that clearly state what they intend to do

File: strelets/adapter/swarm_volumes.go (hunk 2 above)
Comment (netoneko) [reply strelets/adapter/swarm_volumes.go]: Fixed.

**Pull request merged**